client/ayon_maya/plugins/load/load_anim.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+import maya.cmds as cmds
+import ayon_api
+from ayon_maya.api import plugin
+from ayon_core.pipeline.load.utils import get_representation_context
+from ayon_maya.plugins.load.load_reference import ReferenceLoader
+import logging
+import pymel.core as pm
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class AnimLoader(plugin.Loader):
```

Comment:
You can inherit the loader from `plugin.ReferenceLoader` instead. 
Then you dont need to implement things in line 54-56